### DISCUSSION TITLE
Why search in hidden lines?

### DISCUSSION THREAD

**Discussion by Pierre-de-la-Verre** [WinMerge/winmerge#1064] [Ideas]
I have set "View - Diff-Context - 0 lines". Then I start s simple search for a string - it finds something e.g. on "line 4228, col .."
The problem is - the line is hidden and I can see nothing.

Is there a way to search only in displayed lines?

**Comment by sdottaka** [comment]
Unfortunately, for now, WinMerge will also search for hidden lines.
As a workaround, when you copy the whole text, the hidden lines are not copied, so you can paste them into another editor and search for them.

**Reply by Pierre-de-la-Verre** [reply]
> ... for now, ...

So we can hope for the future ;-)
Thanks.
